Rakefile

```diff
@@ -8,9 +8,12 @@ task default: :test
 
 desc 'Run the test stuite'
 Rake::TestTask.new do |t|
+  files = ARGV[1..-1]
+  files = "test/**/*_test.rb" if !files || files.length == 0
```

Comment:
Another option, if you think it reads cleaner: `unless files && files.length > 0`